lib/ProMotion/table/table.rb

```diff
@@ -145,8 +144,7 @@ def create_table_cell(data_cell)
     end
 
     def update_table_data(args = {})
-      # Try and detect if the args param is a NSIndexPath or an array of them
-      args = { index_paths: args } if args.is_a?(NSIndexPath) || (args.is_a?(Array) && array_all_members_of?(args, NSIndexPath))
+      args = { index_paths: args } unless args.is_a?(Hash)
```

Comment:
I don't know if @markrickert sees any issues here or not, but I simplified this line quite a bit. The `is_a?` doesn't work, as stated above, and looking at the method I couldn't see any reason not to do this. Tests all pass fine.